__tests__/nc-news8.test.js

```diff
@@ -12,7 +12,53 @@ describe("PATCH /api/articles/:article_id", () => {
   test("should return 200 status code if sent an object with correct data", () => {
     const newVote = 1;
     const updateVote = { inc_votes: newVote };
-
-    return request(app).patch("/api/articles/4").send(updateVote);
+    return request(app).patch("/api/articles/1").send(updateVote).expect(201);
+  });
+  test("should add votes to an article using article_id", () => {
+    const newVote = 10;
+    const udpateVote = { inc_votes: newVote };
+    return request(app)
+      .get("/api/articles/1")
+      .then(({ body }) => {
+        const { articles } = body;
+        return articles.votes;
+      })
+      .then((currentVotes) => {
+        return request(app)
+          .patch("/api/articles/1")
+          .send(udpateVote)
+          .expect(201)
+          .then(({ body }) => {
+            const { updatedArticle } = body;
+            expect(currentVotes + newVote).toEqual(updatedArticle.votes);
+          });
+      });
+  });
```

Comment:
I like your thinking here but you could just hard code this! You can check the data yourself and expect the votes to be 20 for example

Reply:
I have tried hardcoding, but every time I run the code the votes increment and therefore the test keeps failing. Thats why left it to update dynamically.

Reply:
Implementing a beforeEach to see before each test will help with this :) definitely worth implementing at this stage as it could cause bigger issues later.

Reply:
Thanks, this worked and I have added the hardcoded data.